### PR TITLE
pointing to proper grafana dashboard

### PR DESCRIPTION
**Pull request by agmsb**

### CODE REVIEW

**Comment by sdake** [comment]
@geeknoid this one is stuck - may need to manually merge.

Cheers
-steve

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=
 then
 
 {{< text plain >}}
-http://localhost:3000/dashboard/db/istio-dashboard
+http://localhost:3000/dashboard/db/istio-workload-dashboard
```
Comment (douglas-reid) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: should this be istio-mesh-dashboard?

Comment (sdake) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: @douglas-reid ya - that would be a better generalized dashboard.

Comment (sdake) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: @agmsb image attached for usage in this PR for @douglas-reid's suggestion:

screen shot 2018-09-25 at 8 47 34 pm

**Comment by geeknoid** [comment]
@agmsb Do you want to incorporate the feedback?

**Comment by sdake** [comment]
@geeknoid this PR seems to have stalled.  Given the 1.1 schedule, happy to take iit over - even have a screenshot ready to go.

**Comment by geeknoid** [comment]
@sdake Please do, let's land this.

Thanks.

**Comment by agmsb** [comment]
Apologies for the stall @geeknoid. @sdake have you picked this up? Else I can wrap this up.

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md (hunk 1 above)
Comment (agmsb) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: This image LGTM

Comment (agmsb) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: Updated to istio-mesh-dashboard

**Comment by sdake** [comment]
@agmsb can you rebase if circleci shows a failure?  TX.

Cheers,
-steve

**Comment by agmsb** [comment]
Should be all set @sdake.

**Comment by agmsb** [comment]
Friendly bump, what else is needed?

**Pull request merged**